gcalcli/gcalcli.py

```diff
@@ -1607,6 +1665,9 @@ def main():
         if FLAGS.command == 'list':
             gcal.ListAllCalendars()
 
+        elif FLAGS.command == "colors":
+            gcal.GetColours()
```

Comment:
I suppose that we should leave out the u in the code---mainly since the other usages of "color" in the codebase omit the u.

Reply:
Also, I don't find this function in the change set (or in the existing codebase), am I overlooking something?

Reply:
Sorry - I'd written a new command called `colours` in an earlier commit to query the Google API about for their `colorId` options and left it in there. Will remove it. If it does ever come back, it will do so without the 'u'....